mono/tests/Makefile.am

```diff
@@ -632,7 +673,11 @@ AOT_DISABLED_TESTS=constraints-load.exe
 TEST_CSC_SRC=			\
 	vararg.cs
 
-TEST_IL_SRC=			\
+IL_SRC_MOBILE_STATIC_NOT_SUPPORTED=	\
+	constraints-load.il \
```

Comment:
There's no reason on why constraints-load.il is not supported.
My point is that there's a distinction between not supported and broken. We must make sure we don't confuse the two.